Synthesis failed for nodejs-dataproc

Hello! Autosynth couldn't regenerate nodejs-dataproc. :broken_heart:

Here's the output from running `synth.py`:

```
b'in/python3 -m synthtool --metadata synth.metadata synth.py --\n2020-05-28 04:48:18,580 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py.\nOn branch autosynth-self-2\nnothing to commit, working tree clean\n2020-05-28 04:48:18,707 synthtool [DEBUG] > Ensuring dependencies.\n2020-05-28 04:48:18,712 synthtool [DEBUG] > Cloning googleapis.\n2020-05-28 04:48:18,713 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis\n2020-05-28 04:48:18,716 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:bd4daeabbb1a59b2cde43a47dbfb431eec629d1e54c4dd1ce19220aab3fe8c7d\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-05-28 04:48:19,584 synthtool [DEBUG] > Generating code for: google/cloud/dataproc/v1.\n2020-05-28 04:48:20,957 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py", line 30, in <module>\n    proto_path=f\'/google/cloud/dataproc/{version}\',\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 63, in typescript_library\n    return self._generate_code(service, version, "typescript", **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 192, in _generate_code\n    f"Code generation seemed to succeed, but {output_dir} is empty."\nRuntimeError: Code generation seemed to succeed, but /tmpfs/tmp/tmpthia3cgs is empty.\n2020-05-28 04:48:20,998 autosynth [ERROR] > Synthesis failed\n2020-05-28 04:48:20,998 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-05-28 04:48:21,003 autosynth [DEBUG] > Running: git checkout autosynth-self\nSwitched to branch \'autosynth-self\'\n2020-05-28 04:48:21,007 autosynth [ERROR] > Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n2020-05-28 04:48:21,149 autosynth [DEBUG] > Running: git checkout 6f5a2a4de59410837f7bfeec525d7492dda01dd7\nNote: checking out \'6f5a2a4de59410837f7bfeec525d7492dda01dd7\'.\n\nYou are in \'detached HEAD\' state. You can look around, make experimental\nchanges and commit them, and you can discard any commits you make in this\nstate without impacting any branches by performing another checkout.\n\nIf you want to create a new branch to retain commits you create, you may\ndo so (now or later) by using -b with the checkout command again. Example:\n\n  git checkout -b <new-branch-name>\n\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-05-28 04:48:21,154 autosynth [DEBUG] > Running: git checkout eafa840ceec23b44a5c21670288107c661252711\nPrevious HEAD position was a3a0bf0f BREAKING_CHANGE: Removing TimeSeriesQueryLanguageCondition as an alert condition type. The condition type is unsupported and unused. It was originally added for the Monitoring Query Language Alpha feature. refactor!: Drop support for TimeSeriesQueryLanguageCondition as an alert condition type.\nHEAD is now at eafa840c Use correct resource type for DetachSubscriptionRequest\n2020-05-28 04:48:21,195 autosynth [DEBUG] > Running: git checkout 71b8a272549c06b5768d00fa48d3ae990e871bec\nPrevious HEAD position was be74d3e build: do not fail builds on codecov errors (#528)\nHEAD is now at 71b8a27 docs: update python docs template (#576)\n2020-05-28 04:48:21,208 autosynth [DEBUG] > Running: git branch -f autosynth-84\n2020-05-28 04:48:21,211 autosynth [DEBUG] > Running: git checkout autosynth-84\nSwitched to branch \'autosynth-84\'\n2020-05-28 04:48:21,215 autosynth [INFO] > Running synthtool\n2020-05-28 04:48:21,216 autosynth [INFO] > [\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\n2020-05-28 04:48:21,217 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --\n2020-05-28 04:48:21,414 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py.\nOn branch autosynth-84\nnothing to commit, working tree clean\n2020-05-28 04:48:21,543 synthtool [DEBUG] > Ensuring dependencies.\n2020-05-28 04:48:21,547 synthtool [DEBUG] > Cloning googleapis.\n2020-05-28 04:48:21,548 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis\n2020-05-28 04:48:21,551 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:bd4daeabbb1a59b2cde43a47dbfb431eec629d1e54c4dd1ce19220aab3fe8c7d\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-05-28 04:48:22,395 synthtool [DEBUG] > Generating code for: google/cloud/dataproc/v1.\n2020-05-28 04:48:23,755 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py", line 30, in <module>\n    proto_path=f\'/google/cloud/dataproc/{version}\',\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 63, in typescript_library\n    return self._generate_code(service, version, "typescript", **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 192, in _generate_code\n    f"Code generation seemed to succeed, but {output_dir} is empty."\nRuntimeError: Code generation seemed to succeed, but /tmpfs/tmp/tmp5r517ce9 is empty.\n2020-05-28 04:48:23,794 autosynth [ERROR] > Synthesis failed\n2020-05-28 04:48:23,795 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-05-28 04:48:23,800 autosynth [DEBUG] > Running: git checkout autosynth\nSwitched to branch \'autosynth\'\n2020-05-28 04:48:23,804 autosynth [DEBUG] > Running: git clean -fdx\nRemoving __pycache__/\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main\n    return _inner_main(temp_dir)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main\n    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop\n    synthesize_inner_loop(toolbox, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop\n    synthesizer, len(toolbox.versions) - 1\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch\n    synthesizer.synthesize(synth_log_path, self.environ)\n  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize\n    synth_proc.check_returncode()  # Raise an exception.\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode\n    self.stderr)\nsubprocess.CalledProcessError: Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n'
```

Google internal developers can see the full log [here](http://sponge/fcd31a11-026e-44ae-bc98-9d91fa0c3e53).

Autosynth is still having trouble generating nodejs-dataproc. :sob:

Here's the output from running `synth.py`:

```
b'm synthtool --metadata synth.metadata synth.py --\n2020-05-29 04:53:38,156 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py.\nOn branch autosynth-self-2\nnothing to commit, working tree clean\n2020-05-29 04:53:38,294 synthtool [DEBUG] > Ensuring dependencies.\n2020-05-29 04:53:38,298 synthtool [DEBUG] > Cloning googleapis.\n2020-05-29 04:53:38,299 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis\n2020-05-29 04:53:38,303 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:bd4daeabbb1a59b2cde43a47dbfb431eec629d1e54c4dd1ce19220aab3fe8c7d\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-05-29 04:53:39,228 synthtool [DEBUG] > Generating code for: google/cloud/dataproc/v1.\n2020-05-29 04:53:40,726 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py", line 30, in <module>\n    proto_path=f\'/google/cloud/dataproc/{version}\',\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 66, in typescript_library\n    return self._generate_code(service, version, "typescript", **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 195, in _generate_code\n    f"Code generation seemed to succeed, but {output_dir} is empty."\nRuntimeError: Code generation seemed to succeed, but /tmpfs/tmp/tmp_b_44vc0 is empty.\n2020-05-29 04:53:40,767 autosynth [ERROR] > Synthesis failed\n2020-05-29 04:53:40,768 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-05-29 04:53:40,773 autosynth [DEBUG] > Running: git checkout autosynth-self\nSwitched to branch \'autosynth-self\'\n2020-05-29 04:53:40,778 autosynth [ERROR] > Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n2020-05-29 04:53:40,907 autosynth [DEBUG] > Running: git checkout 6f5a2a4de59410837f7bfeec525d7492dda01dd7\nNote: checking out \'6f5a2a4de59410837f7bfeec525d7492dda01dd7\'.\n\nYou are in \'detached HEAD\' state. You can look around, make experimental\nchanges and commit them, and you can discard any commits you make in this\nstate without impacting any branches by performing another checkout.\n\nIf you want to create a new branch to retain commits you create, you may\ndo so (now or later) by using -b with the checkout command again. Example:\n\n  git checkout -b <new-branch-name>\n\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-05-29 04:53:40,912 autosynth [DEBUG] > Running: git checkout eafa840ceec23b44a5c21670288107c661252711\nPrevious HEAD position was a3a0bf0f BREAKING_CHANGE: Removing TimeSeriesQueryLanguageCondition as an alert condition type. The condition type is unsupported and unused. It was originally added for the Monitoring Query Language Alpha feature. refactor!: Drop support for TimeSeriesQueryLanguageCondition as an alert condition type.\nHEAD is now at eafa840c Use correct resource type for DetachSubscriptionRequest\n2020-05-29 04:53:40,956 autosynth [DEBUG] > Running: git checkout 4e1d2cb79b02d7496b1452f91c518630c207145e\nPrevious HEAD position was be74d3e build: do not fail builds on codecov errors (#528)\nHEAD is now at 4e1d2cb feat: allow custom python versions in noxfile (#585)\n2020-05-29 04:53:40,971 autosynth [DEBUG] > Running: git branch -f autosynth-90\n2020-05-29 04:53:40,975 autosynth [DEBUG] > Running: git checkout autosynth-90\nSwitched to branch \'autosynth-90\'\n2020-05-29 04:53:40,979 autosynth [INFO] > Running synthtool\n2020-05-29 04:53:40,980 autosynth [INFO] > [\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\n2020-05-29 04:53:40,982 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --\n2020-05-29 04:53:41,208 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py.\nOn branch autosynth-90\nnothing to commit, working tree clean\n2020-05-29 04:53:41,347 synthtool [DEBUG] > Ensuring dependencies.\n2020-05-29 04:53:41,352 synthtool [DEBUG] > Cloning googleapis.\n2020-05-29 04:53:41,353 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis\n2020-05-29 04:53:41,356 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:bd4daeabbb1a59b2cde43a47dbfb431eec629d1e54c4dd1ce19220aab3fe8c7d\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-05-29 04:53:42,479 synthtool [DEBUG] > Generating code for: google/cloud/dataproc/v1.\n2020-05-29 04:53:43,915 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py", line 30, in <module>\n    proto_path=f\'/google/cloud/dataproc/{version}\',\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 66, in typescript_library\n    return self._generate_code(service, version, "typescript", **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 195, in _generate_code\n    f"Code generation seemed to succeed, but {output_dir} is empty."\nRuntimeError: Code generation seemed to succeed, but /tmpfs/tmp/tmpv8x_4coh is empty.\n2020-05-29 04:53:43,955 autosynth [ERROR] > Synthesis failed\n2020-05-29 04:53:43,956 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-05-29 04:53:43,961 autosynth [DEBUG] > Running: git checkout autosynth\nSwitched to branch \'autosynth\'\n2020-05-29 04:53:43,965 autosynth [DEBUG] > Running: git clean -fdx\nRemoving __pycache__/\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main\n    return _inner_main(temp_dir)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main\n    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop\n    synthesize_inner_loop(toolbox, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop\n    synthesizer, len(toolbox.versions) - 1\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch\n    synthesizer.synthesize(synth_log_path, self.environ)\n  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize\n    synth_proc.check_returncode()  # Raise an exception.\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode\n    self.stderr)\nsubprocess.CalledProcessError: Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n'
```

Google internal developers can see the full log [here](http://sponge/e6cd6a91-31ba-4151-958e-270932bab16f).

Autosynth is still having trouble generating nodejs-dataproc. :sob:

Here's the output from running `synth.py`:

```
b'nv/bin/python3 -m synthtool --metadata synth.metadata synth.py --\n2020-05-30 04:51:20,295 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py.\nOn branch autosynth-self-2\nnothing to commit, working tree clean\n2020-05-30 04:51:20,437 synthtool [DEBUG] > Ensuring dependencies.\n2020-05-30 04:51:20,443 synthtool [DEBUG] > Cloning googleapis.\n2020-05-30 04:51:20,444 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis\n2020-05-30 04:51:20,448 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:bd4daeabbb1a59b2cde43a47dbfb431eec629d1e54c4dd1ce19220aab3fe8c7d\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-05-30 04:51:21,366 synthtool [DEBUG] > Generating code for: google/cloud/dataproc/v1.\n2020-05-30 04:51:22,840 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py", line 30, in <module>\n    proto_path=f\'/google/cloud/dataproc/{version}\',\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 66, in typescript_library\n    return self._generate_code(service, version, "typescript", **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 195, in _generate_code\n    f"Code generation seemed to succeed, but {output_dir} is empty."\nRuntimeError: Code generation seemed to succeed, but /tmpfs/tmp/tmprvc2v478 is empty.\n2020-05-30 04:51:22,891 autosynth [ERROR] > Synthesis failed\n2020-05-30 04:51:22,892 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-05-30 04:51:22,898 autosynth [DEBUG] > Running: git checkout autosynth-self\nSwitched to branch \'autosynth-self\'\n2020-05-30 04:51:22,904 autosynth [ERROR] > Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n2020-05-30 04:51:23,044 autosynth [DEBUG] > Running: git checkout 6f5a2a4de59410837f7bfeec525d7492dda01dd7\nNote: checking out \'6f5a2a4de59410837f7bfeec525d7492dda01dd7\'.\n\nYou are in \'detached HEAD\' state. You can look around, make experimental\nchanges and commit them, and you can discard any commits you make in this\nstate without impacting any branches by performing another checkout.\n\nIf you want to create a new branch to retain commits you create, you may\ndo so (now or later) by using -b with the checkout command again. Example:\n\n  git checkout -b <new-branch-name>\n\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-05-30 04:51:23,049 autosynth [DEBUG] > Running: git checkout 9309ee73424d2798a2557d607dfb3e91c8a433fb\nPrevious HEAD position was a3a0bf0f BREAKING_CHANGE: Removing TimeSeriesQueryLanguageCondition as an alert condition type. The condition type is unsupported and unused. It was originally added for the Monitoring Query Language Alpha feature. refactor!: Drop support for TimeSeriesQueryLanguageCondition as an alert condition type.\nHEAD is now at 9309ee73 chore: enable gapicv2 for oslogin/v1beta1 API\n2020-05-30 04:51:23,096 autosynth [DEBUG] > Running: git checkout cb3c683e958a4b5c016bb3734436fc1cb887eb7b\nPrevious HEAD position was be74d3e build: do not fail builds on codecov errors (#528)\nHEAD is now at cb3c683 chore: fix README instructions for java (#592)\n2020-05-30 04:51:23,112 autosynth [DEBUG] > Running: git branch -f autosynth-97\n2020-05-30 04:51:23,117 autosynth [DEBUG] > Running: git checkout autosynth-97\nSwitched to branch \'autosynth-97\'\n2020-05-30 04:51:23,123 autosynth [INFO] > Running synthtool\n2020-05-30 04:51:23,123 autosynth [INFO] > [\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\n2020-05-30 04:51:23,126 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --\n2020-05-30 04:51:23,346 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py.\nOn branch autosynth-97\nnothing to commit, working tree clean\n2020-05-30 04:51:23,487 synthtool [DEBUG] > Ensuring dependencies.\n2020-05-30 04:51:23,493 synthtool [DEBUG] > Cloning googleapis.\n2020-05-30 04:51:23,495 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis\n2020-05-30 04:51:23,499 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:bd4daeabbb1a59b2cde43a47dbfb431eec629d1e54c4dd1ce19220aab3fe8c7d\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-05-30 04:51:24,443 synthtool [DEBUG] > Generating code for: google/cloud/dataproc/v1.\n2020-05-30 04:51:25,858 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py", line 30, in <module>\n    proto_path=f\'/google/cloud/dataproc/{version}\',\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 66, in typescript_library\n    return self._generate_code(service, version, "typescript", **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 195, in _generate_code\n    f"Code generation seemed to succeed, but {output_dir} is empty."\nRuntimeError: Code generation seemed to succeed, but /tmpfs/tmp/tmpjlymcjmz is empty.\n2020-05-30 04:51:25,908 autosynth [ERROR] > Synthesis failed\n2020-05-30 04:51:25,908 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-05-30 04:51:25,916 autosynth [DEBUG] > Running: git checkout autosynth\nSwitched to branch \'autosynth\'\n2020-05-30 04:51:25,921 autosynth [DEBUG] > Running: git clean -fdx\nRemoving __pycache__/\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main\n    return _inner_main(temp_dir)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main\n    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop\n    synthesize_inner_loop(toolbox, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop\n    synthesizer, len(toolbox.versions) - 1\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch\n    synthesizer.synthesize(synth_log_path, self.environ)\n  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize\n    synth_proc.check_returncode()  # Raise an exception.\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode\n    self.stderr)\nsubprocess.CalledProcessError: Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n'
```

Google internal developers can see the full log [here](http://sponge/1d5e60e7-140e-4e3a-a1d3-c10212a4b176).

Autosynth is still having trouble generating nodejs-dataproc. :sob:

Here's the output from running `synth.py`:

```
b'nv/bin/python3 -m synthtool --metadata synth.metadata synth.py --\n2020-05-31 04:51:08,158 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py.\nOn branch autosynth-self-2\nnothing to commit, working tree clean\n2020-05-31 04:51:08,291 synthtool [DEBUG] > Ensuring dependencies.\n2020-05-31 04:51:08,296 synthtool [DEBUG] > Cloning googleapis.\n2020-05-31 04:51:08,297 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis\n2020-05-31 04:51:08,301 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:bd4daeabbb1a59b2cde43a47dbfb431eec629d1e54c4dd1ce19220aab3fe8c7d\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-05-31 04:51:09,185 synthtool [DEBUG] > Generating code for: google/cloud/dataproc/v1.\n2020-05-31 04:51:10,621 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py", line 30, in <module>\n    proto_path=f\'/google/cloud/dataproc/{version}\',\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 66, in typescript_library\n    return self._generate_code(service, version, "typescript", **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 195, in _generate_code\n    f"Code generation seemed to succeed, but {output_dir} is empty."\nRuntimeError: Code generation seemed to succeed, but /tmpfs/tmp/tmpugt5zq8s is empty.\n2020-05-31 04:51:10,665 autosynth [ERROR] > Synthesis failed\n2020-05-31 04:51:10,665 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-05-31 04:51:10,671 autosynth [DEBUG] > Running: git checkout autosynth-self\nSwitched to branch \'autosynth-self\'\n2020-05-31 04:51:10,675 autosynth [ERROR] > Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n2020-05-31 04:51:10,818 autosynth [DEBUG] > Running: git checkout 6f5a2a4de59410837f7bfeec525d7492dda01dd7\nNote: checking out \'6f5a2a4de59410837f7bfeec525d7492dda01dd7\'.\n\nYou are in \'detached HEAD\' state. You can look around, make experimental\nchanges and commit them, and you can discard any commits you make in this\nstate without impacting any branches by performing another checkout.\n\nIf you want to create a new branch to retain commits you create, you may\ndo so (now or later) by using -b with the checkout command again. Example:\n\n  git checkout -b <new-branch-name>\n\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-05-31 04:51:10,823 autosynth [DEBUG] > Running: git checkout 9309ee73424d2798a2557d607dfb3e91c8a433fb\nPrevious HEAD position was a3a0bf0f BREAKING_CHANGE: Removing TimeSeriesQueryLanguageCondition as an alert condition type. The condition type is unsupported and unused. It was originally added for the Monitoring Query Language Alpha feature. refactor!: Drop support for TimeSeriesQueryLanguageCondition as an alert condition type.\nHEAD is now at 9309ee73 chore: enable gapicv2 for oslogin/v1beta1 API\n2020-05-31 04:51:10,867 autosynth [DEBUG] > Running: git checkout cb3c683e958a4b5c016bb3734436fc1cb887eb7b\nPrevious HEAD position was be74d3e build: do not fail builds on codecov errors (#528)\nHEAD is now at cb3c683 chore: fix README instructions for java (#592)\n2020-05-31 04:51:10,882 autosynth [DEBUG] > Running: git branch -f autosynth-97\n2020-05-31 04:51:10,886 autosynth [DEBUG] > Running: git checkout autosynth-97\nSwitched to branch \'autosynth-97\'\n2020-05-31 04:51:10,891 autosynth [INFO] > Running synthtool\n2020-05-31 04:51:10,892 autosynth [INFO] > [\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\n2020-05-31 04:51:10,894 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --\n2020-05-31 04:51:11,099 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py.\nOn branch autosynth-97\nnothing to commit, working tree clean\n2020-05-31 04:51:11,231 synthtool [DEBUG] > Ensuring dependencies.\n2020-05-31 04:51:11,236 synthtool [DEBUG] > Cloning googleapis.\n2020-05-31 04:51:11,237 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis\n2020-05-31 04:51:11,240 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:bd4daeabbb1a59b2cde43a47dbfb431eec629d1e54c4dd1ce19220aab3fe8c7d\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-05-31 04:51:12,100 synthtool [DEBUG] > Generating code for: google/cloud/dataproc/v1.\n2020-05-31 04:51:13,501 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py", line 30, in <module>\n    proto_path=f\'/google/cloud/dataproc/{version}\',\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 66, in typescript_library\n    return self._generate_code(service, version, "typescript", **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 195, in _generate_code\n    f"Code generation seemed to succeed, but {output_dir} is empty."\nRuntimeError: Code generation seemed to succeed, but /tmpfs/tmp/tmpo_6dorir is empty.\n2020-05-31 04:51:13,545 autosynth [ERROR] > Synthesis failed\n2020-05-31 04:51:13,546 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-05-31 04:51:13,551 autosynth [DEBUG] > Running: git checkout autosynth\nSwitched to branch \'autosynth\'\n2020-05-31 04:51:13,556 autosynth [DEBUG] > Running: git clean -fdx\nRemoving __pycache__/\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main\n    return _inner_main(temp_dir)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main\n    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop\n    synthesize_inner_loop(toolbox, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop\n    synthesizer, len(toolbox.versions) - 1\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch\n    synthesizer.synthesize(synth_log_path, self.environ)\n  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize\n    synth_proc.check_returncode()  # Raise an exception.\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode\n    self.stderr)\nsubprocess.CalledProcessError: Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n'
```

Google internal developers can see the full log [here](http://sponge/caac78a0-8efc-45a0-9271-644becae9ccb).

Autosynth is still having trouble generating nodejs-dataproc. :sob:

Here's the output from running `synth.py`:

```
b'nv/bin/python3 -m synthtool --metadata synth.metadata synth.py --\n2020-06-01 04:53:15,195 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py.\nOn branch autosynth-self-2\nnothing to commit, working tree clean\n2020-06-01 04:53:15,332 synthtool [DEBUG] > Ensuring dependencies.\n2020-06-01 04:53:15,336 synthtool [DEBUG] > Cloning googleapis.\n2020-06-01 04:53:15,337 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis\n2020-06-01 04:53:15,340 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:bd4daeabbb1a59b2cde43a47dbfb431eec629d1e54c4dd1ce19220aab3fe8c7d\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-06-01 04:53:16,238 synthtool [DEBUG] > Generating code for: google/cloud/dataproc/v1.\n2020-06-01 04:53:17,675 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py", line 30, in <module>\n    proto_path=f\'/google/cloud/dataproc/{version}\',\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 66, in typescript_library\n    return self._generate_code(service, version, "typescript", **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 195, in _generate_code\n    f"Code generation seemed to succeed, but {output_dir} is empty."\nRuntimeError: Code generation seemed to succeed, but /tmpfs/tmp/tmpntosuoba is empty.\n2020-06-01 04:53:17,712 autosynth [ERROR] > Synthesis failed\n2020-06-01 04:53:17,712 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-06-01 04:53:17,720 autosynth [DEBUG] > Running: git checkout autosynth-self\nSwitched to branch \'autosynth-self\'\n2020-06-01 04:53:17,724 autosynth [ERROR] > Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n2020-06-01 04:53:17,871 autosynth [DEBUG] > Running: git checkout 6f5a2a4de59410837f7bfeec525d7492dda01dd7\nNote: checking out \'6f5a2a4de59410837f7bfeec525d7492dda01dd7\'.\n\nYou are in \'detached HEAD\' state. You can look around, make experimental\nchanges and commit them, and you can discard any commits you make in this\nstate without impacting any branches by performing another checkout.\n\nIf you want to create a new branch to retain commits you create, you may\ndo so (now or later) by using -b with the checkout command again. Example:\n\n  git checkout -b <new-branch-name>\n\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-06-01 04:53:17,876 autosynth [DEBUG] > Running: git checkout 9309ee73424d2798a2557d607dfb3e91c8a433fb\nPrevious HEAD position was a3a0bf0f BREAKING_CHANGE: Removing TimeSeriesQueryLanguageCondition as an alert condition type. The condition type is unsupported and unused. It was originally added for the Monitoring Query Language Alpha feature. refactor!: Drop support for TimeSeriesQueryLanguageCondition as an alert condition type.\nHEAD is now at 9309ee73 chore: enable gapicv2 for oslogin/v1beta1 API\n2020-06-01 04:53:17,919 autosynth [DEBUG] > Running: git checkout cb3c683e958a4b5c016bb3734436fc1cb887eb7b\nPrevious HEAD position was be74d3e build: do not fail builds on codecov errors (#528)\nHEAD is now at cb3c683 chore: fix README instructions for java (#592)\n2020-06-01 04:53:17,934 autosynth [DEBUG] > Running: git branch -f autosynth-97\n2020-06-01 04:53:17,937 autosynth [DEBUG] > Running: git checkout autosynth-97\nSwitched to branch \'autosynth-97\'\n2020-06-01 04:53:17,941 autosynth [INFO] > Running synthtool\n2020-06-01 04:53:17,942 autosynth [INFO] > [\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\n2020-06-01 04:53:17,943 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --\n2020-06-01 04:53:18,159 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py.\nOn branch autosynth-97\nnothing to commit, working tree clean\n2020-06-01 04:53:18,297 synthtool [DEBUG] > Ensuring dependencies.\n2020-06-01 04:53:18,302 synthtool [DEBUG] > Cloning googleapis.\n2020-06-01 04:53:18,303 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis\n2020-06-01 04:53:18,306 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:bd4daeabbb1a59b2cde43a47dbfb431eec629d1e54c4dd1ce19220aab3fe8c7d\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-06-01 04:53:19,181 synthtool [DEBUG] > Generating code for: google/cloud/dataproc/v1.\n2020-06-01 04:53:20,565 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py", line 30, in <module>\n    proto_path=f\'/google/cloud/dataproc/{version}\',\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 66, in typescript_library\n    return self._generate_code(service, version, "typescript", **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 195, in _generate_code\n    f"Code generation seemed to succeed, but {output_dir} is empty."\nRuntimeError: Code generation seemed to succeed, but /tmpfs/tmp/tmpe0cn6vze is empty.\n2020-06-01 04:53:20,601 autosynth [ERROR] > Synthesis failed\n2020-06-01 04:53:20,601 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-06-01 04:53:20,606 autosynth [DEBUG] > Running: git checkout autosynth\nSwitched to branch \'autosynth\'\n2020-06-01 04:53:20,610 autosynth [DEBUG] > Running: git clean -fdx\nRemoving __pycache__/\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main\n    return _inner_main(temp_dir)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main\n    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop\n    synthesize_inner_loop(toolbox, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop\n    synthesizer, len(toolbox.versions) - 1\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch\n    synthesizer.synthesize(synth_log_path, self.environ)\n  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize\n    synth_proc.check_returncode()  # Raise an exception.\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode\n    self.stderr)\nsubprocess.CalledProcessError: Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n'
```

Google internal developers can see the full log [here](http://sponge/ce173868-0f59-4462-8945-36e31f9c6542).

Autosynth is still having trouble generating nodejs-dataproc. :sob:

Here's the output from running `synth.py`:

```
b' branch autosynth-self-2\nnothing to commit, working tree clean\n2020-06-02 04:49:33,176 synthtool [DEBUG] > Ensuring dependencies.\n2020-06-02 04:49:33,181 synthtool [DEBUG] > Cloning googleapis.\n2020-06-02 04:49:33,181 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis\n2020-06-02 04:49:33,185 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:bd4daeabbb1a59b2cde43a47dbfb431eec629d1e54c4dd1ce19220aab3fe8c7d\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-06-02 04:49:34,048 synthtool [DEBUG] > Generating code for: google/cloud/dataproc/v1.\n2020-06-02 04:49:35,430 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py", line 30, in <module>\n    proto_path=f\'/google/cloud/dataproc/{version}\',\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 66, in typescript_library\n    return self._generate_code(service, version, "typescript", **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 195, in _generate_code\n    f"Code generation seemed to succeed, but {output_dir} is empty."\nRuntimeError: Code generation seemed to succeed, but /tmpfs/tmp/tmpcfw9kvze is empty.\n2020-06-02 04:49:35,467 autosynth [ERROR] > Synthesis failed\n2020-06-02 04:49:35,467 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-06-02 04:49:35,473 autosynth [DEBUG] > Running: git checkout autosynth-self\nSwitched to branch \'autosynth-self\'\n2020-06-02 04:49:35,477 autosynth [ERROR] > Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n2020-06-02 04:49:35,609 autosynth [DEBUG] > Running: git checkout 6f5a2a4de59410837f7bfeec525d7492dda01dd7\nNote: checking out \'6f5a2a4de59410837f7bfeec525d7492dda01dd7\'.\n\nYou are in \'detached HEAD\' state. You can look around, make experimental\nchanges and commit them, and you can discard any commits you make in this\nstate without impacting any branches by performing another checkout.\n\nIf you want to create a new branch to retain commits you create, you may\ndo so (now or later) by using -b with the checkout command again. Example:\n\n  git checkout -b <new-branch-name>\n\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-06-02 04:49:35,614 autosynth [DEBUG] > Running: git checkout 907754d70ec714ab0e8282fe6ff4988221c64130\nPrevious HEAD position was a3a0bf0f BREAKING_CHANGE: Removing TimeSeriesQueryLanguageCondition as an alert condition type. The condition type is unsupported and unused. It was originally added for the Monitoring Query Language Alpha feature. refactor!: Drop support for TimeSeriesQueryLanguageCondition as an alert condition type.\nHEAD is now at 907754d7 - Add CreateCapacityCommitment method to BigQuery Reservation v1beta1. - Update missing child_type annotations for v1 and v1beta1 proto annotations. - Revert name of bi_reservation back to reservation for v1beta1 service.\n2020-06-02 04:49:35,674 autosynth [DEBUG] > Running: git checkout cb3c683e958a4b5c016bb3734436fc1cb887eb7b\nPrevious HEAD position was be74d3e build: do not fail builds on codecov errors (#528)\nHEAD is now at cb3c683 chore: fix README instructions for java (#592)\n2020-06-02 04:49:35,689 autosynth [DEBUG] > Running: git branch -f autosynth-100\n2020-06-02 04:49:35,693 autosynth [DEBUG] > Running: git checkout autosynth-100\nSwitched to branch \'autosynth-100\'\n2020-06-02 04:49:35,697 autosynth [INFO] > Running synthtool\n2020-06-02 04:49:35,698 autosynth [INFO] > [\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\n2020-06-02 04:49:35,699 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --\n2020-06-02 04:49:35,896 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py.\nOn branch autosynth-100\nnothing to commit, working tree clean\n2020-06-02 04:49:36,025 synthtool [DEBUG] > Ensuring dependencies.\n2020-06-02 04:49:36,030 synthtool [DEBUG] > Cloning googleapis.\n2020-06-02 04:49:36,031 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis\n2020-06-02 04:49:36,034 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:bd4daeabbb1a59b2cde43a47dbfb431eec629d1e54c4dd1ce19220aab3fe8c7d\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-06-02 04:49:36,894 synthtool [DEBUG] > Generating code for: google/cloud/dataproc/v1.\n2020-06-02 04:49:38,231 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py", line 30, in <module>\n    proto_path=f\'/google/cloud/dataproc/{version}\',\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 66, in typescript_library\n    return self._generate_code(service, version, "typescript", **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 195, in _generate_code\n    f"Code generation seemed to succeed, but {output_dir} is empty."\nRuntimeError: Code generation seemed to succeed, but /tmpfs/tmp/tmpqwu0flxj is empty.\n2020-06-02 04:49:38,270 autosynth [ERROR] > Synthesis failed\n2020-06-02 04:49:38,270 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-06-02 04:49:38,275 autosynth [DEBUG] > Running: git checkout autosynth\nSwitched to branch \'autosynth\'\n2020-06-02 04:49:38,280 autosynth [DEBUG] > Running: git clean -fdx\nRemoving __pycache__/\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main\n    return _inner_main(temp_dir)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main\n    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop\n    synthesize_inner_loop(toolbox, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop\n    synthesizer, len(toolbox.versions) - 1\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch\n    synthesizer.synthesize(synth_log_path, self.environ)\n  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize\n    synth_proc.check_returncode()  # Raise an exception.\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode\n    self.stderr)\nsubprocess.CalledProcessError: Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n'
```

Google internal developers can see the full log [here](http://sponge/88eaf4b5-788f-4253-9cae-313d8d1efc4d).

Autosynth is still having trouble generating nodejs-dataproc. :sob:

Here's the output from running `synth.py`:

```
b'thub/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --\n2020-06-03 05:21:11,468 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py.\nOn branch autosynth-self-2\nnothing to commit, working tree clean\n2020-06-03 05:21:11,608 synthtool [DEBUG] > Ensuring dependencies.\n2020-06-03 05:21:11,614 synthtool [DEBUG] > Cloning googleapis.\n2020-06-03 05:21:11,615 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis\n2020-06-03 05:21:11,619 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:bd4daeabbb1a59b2cde43a47dbfb431eec629d1e54c4dd1ce19220aab3fe8c7d\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-06-03 05:21:12,582 synthtool [DEBUG] > Generating code for: google/cloud/dataproc/v1.\n2020-06-03 05:21:14,078 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py", line 30, in <module>\n    proto_path=f\'/google/cloud/dataproc/{version}\',\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 66, in typescript_library\n    return self._generate_code(service, version, "typescript", **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 195, in _generate_code\n    f"Code generation seemed to succeed, but {output_dir} is empty."\nRuntimeError: Code generation seemed to succeed, but /tmpfs/tmp/tmpuv2o889e is empty.\n2020-06-03 05:21:14,128 autosynth [ERROR] > Synthesis failed\n2020-06-03 05:21:14,128 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-06-03 05:21:14,133 autosynth [DEBUG] > Running: git checkout autosynth-self\nSwitched to branch \'autosynth-self\'\n2020-06-03 05:21:14,139 autosynth [ERROR] > Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n2020-06-03 05:21:14,305 autosynth [DEBUG] > Running: git checkout 6f5a2a4de59410837f7bfeec525d7492dda01dd7\nNote: checking out \'6f5a2a4de59410837f7bfeec525d7492dda01dd7\'.\n\nYou are in \'detached HEAD\' state. You can look around, make experimental\nchanges and commit them, and you can discard any commits you make in this\nstate without impacting any branches by performing another checkout.\n\nIf you want to create a new branch to retain commits you create, you may\ndo so (now or later) by using -b with the checkout command again. Example:\n\n  git checkout -b <new-branch-name>\n\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-06-03 05:21:14,310 autosynth [DEBUG] > Running: git checkout d5fe42c39cd35f95131a0267314ae108ab1bef8d\nPrevious HEAD position was a3a0bf0f BREAKING_CHANGE: Removing TimeSeriesQueryLanguageCondition as an alert condition type. The condition type is unsupported and unused. It was originally added for the Monitoring Query Language Alpha feature. refactor!: Drop support for TimeSeriesQueryLanguageCondition as an alert condition type.\nHEAD is now at d5fe42c3 firestore: add BatchWrite\n2020-06-03 05:21:14,376 autosynth [DEBUG] > Running: git checkout cb3c683e958a4b5c016bb3734436fc1cb887eb7b\nPrevious HEAD position was be74d3e build: do not fail builds on codecov errors (#528)\nHEAD is now at cb3c683 chore: fix README instructions for java (#592)\n2020-06-03 05:21:14,392 autosynth [DEBUG] > Running: git branch -f autosynth-106\n2020-06-03 05:21:14,396 autosynth [DEBUG] > Running: git checkout autosynth-106\nSwitched to branch \'autosynth-106\'\n2020-06-03 05:21:14,401 autosynth [INFO] > Running synthtool\n2020-06-03 05:21:14,401 autosynth [INFO] > [\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\n2020-06-03 05:21:14,403 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --\n2020-06-03 05:21:14,619 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py.\nOn branch autosynth-106\nnothing to commit, working tree clean\n2020-06-03 05:21:14,762 synthtool [DEBUG] > Ensuring dependencies.\n2020-06-03 05:21:14,768 synthtool [DEBUG] > Cloning googleapis.\n2020-06-03 05:21:14,769 synthtool [DEBUG] > Using precloned repo /home/kbuilder/.cache/synthtool/googleapis\n2020-06-03 05:21:14,773 synthtool [DEBUG] > Pulling Docker image: gapic-generator-typescript:latest\nlatest: Pulling from gapic-images/gapic-generator-typescript\nDigest: sha256:bd4daeabbb1a59b2cde43a47dbfb431eec629d1e54c4dd1ce19220aab3fe8c7d\nStatus: Image is up to date for gcr.io/gapic-images/gapic-generator-typescript:latest\n2020-06-03 05:21:15,696 synthtool [DEBUG] > Generating code for: google/cloud/dataproc/v1.\n2020-06-03 05:21:17,098 synthtool [DEBUG] > Wrote metadata to synth.metadata.\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__\n    return self.main(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main\n    rv = self.invoke(ctx)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke\n    return ctx.invoke(self.callback, **ctx.params)\n  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke\n    return callback(*args, **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main\n    spec.loader.exec_module(synth_module)  # type: ignore\n  File "<frozen importlib._bootstrap_external>", line 678, in exec_module\n  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed\n  File "/home/kbuilder/.cache/synthtool/nodejs-dataproc/synth.py", line 30, in <module>\n    proto_path=f\'/google/cloud/dataproc/{version}\',\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 66, in typescript_library\n    return self._generate_code(service, version, "typescript", **kwargs)\n  File "/tmpfs/src/github/synthtool/synthtool/gcp/gapic_microgenerator.py", line 195, in _generate_code\n    f"Code generation seemed to succeed, but {output_dir} is empty."\nRuntimeError: Code generation seemed to succeed, but /tmpfs/tmp/tmpyc6jusq4 is empty.\n2020-06-03 05:21:17,150 autosynth [ERROR] > Synthesis failed\n2020-06-03 05:21:17,150 autosynth [DEBUG] > Running: git reset --hard HEAD\nHEAD is now at 6f5a2a4 fix(deps): update dependency @google-cloud/storage to v5 (#380)\n2020-06-03 05:21:17,156 autosynth [DEBUG] > Running: git checkout autosynth\nSwitched to branch \'autosynth\'\n2020-06-03 05:21:17,161 autosynth [DEBUG] > Running: git clean -fdx\nRemoving __pycache__/\nTraceback (most recent call last):\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 193, in _run_module_as_main\n    "__main__", mod_spec)\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/runpy.py", line 85, in _run_code\n    exec(code, run_globals)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 615, in <module>\n    main()\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 476, in main\n    return _inner_main(temp_dir)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 595, in _inner_main\n    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 371, in synthesize_loop\n    synthesize_inner_loop(toolbox, synthesizer)\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 381, in synthesize_inner_loop\n    synthesizer, len(toolbox.versions) - 1\n  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 266, in synthesize_version_in_new_branch\n    synthesizer.synthesize(synth_log_path, self.environ)\n  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 119, in synthesize\n    synth_proc.check_returncode()  # Raise an exception.\n  File "/home/kbuilder/.pyenv/versions/3.6.9/lib/python3.6/subprocess.py", line 389, in check_returncode\n    self.stderr)\nsubprocess.CalledProcessError: Command \'[\'/tmpfs/src/github/synthtool/env/bin/python3\', \'-m\', \'synthtool\', \'--metadata\', \'synth.metadata\', \'synth.py\', \'--\']\' returned non-zero exit status 1.\n'
```

Google internal developers can see the full log [here](http://sponge/041f5df7-f5d3-4b2a-9ede-0752bf41c185).

--- a/protos/google/cloud/dataproc/v1/jobs.proto
+++ b/protos/google/cloud/dataproc/v1/jobs.proto
@@ -435,8 +435,9 @@ message SparkRJob {
 
 // A Dataproc job for running [Presto](https://prestosql.io/) queries.
 // **IMPORTANT**: The [Dataproc Presto Optional
-// Component](/dataproc/docs/concepts/components/presto) must be enabled when
-// the cluster is created to submit a Presto job to the cluster.
+// Component](https://cloud.google.com/dataproc/docs/concepts/components/presto)
+// must be enabled when the cluster is created to submit a Presto job to the
+// cluster.
 message PrestoJob {
   // Required. The sequence of Presto queries to execute, specified as
   // either an HCFS file URI or as a list of queries.

--- a/protos/google/cloud/dataproc/v1beta2/jobs.proto
+++ b/protos/google/cloud/dataproc/v1beta2/jobs.proto
@@ -435,8 +435,9 @@ message SparkRJob {
 
 // A Dataproc job for running [Presto](https://prestosql.io/) queries.
 // **IMPORTANT**: The [Dataproc Presto Optional
-// Component](/dataproc/docs/concepts/components/presto) must be enabled when
-// the cluster is created to submit a Presto job to the cluster.
+// Component](https://cloud.google.com/dataproc/docs/concepts/components/presto)
+// must be enabled when the cluster is created to submit a Presto job to the
+// cluster.
 message PrestoJob {
   // Required. The sequence of Presto queries to execute, specified as
   // either an HCFS file URI or as a list of queries.

--- a/synth.metadata
+++ b/synth.metadata
@@ -3,23 +3,23 @@
     {
       "git": {
         "name": ".",
-        "remote": "https://github.com/googleapis/nodejs-dataproc.git",
-        "sha": "4b84de4a54ea4ac3cbede49ed723480d0d69106d"
+        "remote": "git@github.com:googleapis/nodejs-dataproc.git",
+        "sha": "6f5a2a4de59410837f7bfeec525d7492dda01dd7"
       }
     },
     {
       "git": {
         "name": "googleapis",
         "remote": "https://github.com/googleapis/googleapis.git",
-        "sha": "a3a0bf0f6291d69f2ff3df7fcd63d28ee20ac727",
-        "internalRef": "310060413"
+        "sha": "c1fae183ddeef0c59538863eac611fd679d1b7fb",
+        "internalRef": "314634470"
       }
     },
     {
       "git": {
         "name": "synthtool",
         "remote": "https://github.com/googleapis/synthtool.git",
-        "sha": "be74d3e532faa47eb59f1a0eaebde0860d1d8ab4"
+        "sha": "8b65daa222d193b689279162781baf0aa1f0ffd2"
       }
     }
   ],

--- a/synth.py
+++ b/synth.py
@@ -26,6 +26,7 @@ for version in versions:
             "grpc-service-config": f"google/cloud/dataproc/{version}/dataproc_grpc_service_config.json",
             "package-name": "@google-cloud/dataproc",
             "main-service": "dataproc",
+            "validation": "false",
         },
         proto_path=f'/google/cloud/dataproc/{version}',
     )

